dashboard/config/foorm/forms/surveys/pd/workshop_csf_intro_post.0.json

```diff
@@ -7,115 +7,80 @@
         {
           "type": "html",
           "name": "intro_text",
-          "html": "Please note that the facilitator of your session will see your responses. Your name and email will not be shared with your facilitator to protect your anonymity, however, it’s possible your facilitator could identify you from your written responses."
-        },
-        {
-          "type": "library_item",
-          "library_name": "surveys/pd/workshop_elements",
-          "library_version": 0,
-          "name": "consent_survey"
+          "html": "<p><strong>Well done, you've completed the Code.org CS Fundamentals Intro workshop!</strong></p><p>This survey asks questions about your experience today. Your feedback is crucial to ensuring we best serve educators through continual program improvement.</p><p><strong>Privacy and Confidentiality</strong></p><ul><li>Your individual responses here are completely <strong>confidential</strong>.</li><li>Your identity will <strong>never</strong> be revealed publicly or used to tie any responses back to you personally.</li><li>We may share <strong>de-identified</strong>, aggregate data (representing all responses) publicly.</li><li>Your <strong>de-identified</strong> responses will be used by Code.org, our Regional Partners, and our facilitators for program improvement.</li></ul>"
         },
+
         {
-          "type": "radiogroup",
-          "name": "will_teach_csd_next_year",
-          "title": "Will you be teaching/are you teaching CS Fundamentals 5-day Summer in the next school year? ",
-          "isRequired": true,
-          "choices": [
+          "type": "matrix",
+          "name": "overall_success",
+          "title": "Overall",
+          "description": "How much do you agree or disagree with the following statements about the workshop overall?",
+          "columns": [
             {
-              "value": "yes",
-              "text": "Yes"
+              "value": "strongly_disagree",
```

Comment:
for matrices/other questions we may want to roll up, can we move to using numeric values? That way we can roll them up

Reply:
Good call.